src/lib.rs

```diff
@@ -133,6 +141,7 @@ impl Default for RouterBuilder {
                 .map(|td| td.into_path())
                 .unwrap_or_default(),
             default_host: String::from("localhost:3000"),
+            http_concurrency: Some(1),
```

Comment:
Because this is an option, it should probably be `None` as the default we unwrap to is `1`

Reply:
done